### PR TITLE
React components

### PR DESCRIPTION
**Pull request by TemesgenA**
## Features Implemented
- display the list of books
- display a form for adding a book (no functionality yet)
- Add React Router and set two Routes and Links for the app's navigation

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,39 +1,15 @@
...
+  gap: 2em;
 
-  to {
-    transform: rotate(360deg);
-  }
+  /* margin: 0 2.5rem 4.063rem 0;
+  padding: 0 0 49.625rem;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 var(--black-5);
+  background-color: #fafafa; */
```
Comment (Deepakdanger) [inline src/App.css]: - [ ] Kindly remove the commented code.

File: src/components/BookForm.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
```
Comment (Deepakdanger) [inline src/components/BookForm.js]: - [ ] Kindly remove this & solve it manually.

File: src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+const Header = () => (
+  <div className="Header">
+    <div className="logo">
+      <h1>Bookstore CMS</h1>
+    </div>
+    <nav className="nav">
+      <ul className="nav-links">
+        <li className="books">BOOKS</li>
+        <li className="catagories">CATEGORIES</li>
```
Comment (Deepakdanger) [inline src/components/Header.js:12]: - [ ] Categories
* should display only a button with text "Check status".
* After clicking on the `CATEGORIES` it should display "Check status".

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @TemesgenA , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
- [ ] README file should be descriptive

* All main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in a student's repo
- Kindly fill in all the necessary details.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (TemesgenA) [reply src/App.css]: Thank you.
The issue has been resolved.

File: src/components/BookForm.js (hunk 2 above)
Comment (TemesgenA) [reply src/components/BookForm.js]: Thank you.
The issue has been resolved.

**Review by DammyShittu — APPROVED** [review]
Hi @TemesgenA,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**